### PR TITLE
Update Statistics UITests and screenshots

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Update UITests and screenshots for statistics after adding the Pandemic radar tile

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14084

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift [hunk 1]
```diff
@@ -27,6 +27,14 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				#if DEBUG
+				if isUITesting {
+					var mockedPandemicLinkCardModel = SAP_Internal_Stats_LinkCard()
+					mockedPandemicLinkCardModel.header.cardID = 12
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift:33]: You can use the existing mock `SAP_Internal_Stats_LinkCard.mock(cardID:updatedAt:url:)` from `SAP_Internal_Stats_LinkCard+Mock.swift`. You only have to set the check for UI test target :)

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift [hunk 2]
```diff
@@ -27,6 +27,14 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				#if DEBUG
+				if isUITesting {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift:31]: Can we please wrap this into a function something like setupMockLinkCards and call it on line 54?

**Review by flxschmidt — CHANGES_REQUESTED** [review]
Looks great! Just one suggestion.

File: src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift [hunk 3]
```diff
@@ -389,6 +433,12 @@ class ENAUITests_01b_Statistics: CWATestCase {
 
 	// MARK: - Private
 	
+	private func cardPandemicRadatInfoScreenTest(_ title0: String) {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift:436]:
```suggestion
	private func cardPandemicRadarInfoScreenTest(_ title0: String) {
```

Comment (flxschmidt) [reply src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift:436]: Naveed has eagle eyes!!!! 8)

File: src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift [hunk 4]
```diff
@@ -221,10 +253,14 @@ class ENAUITests_01b_Statistics: CWATestCase {
 			app.staticTexts[title2].swipeLeft()
 
 			cardCombinedIncidencesInfoScreenTest(title1)
-			app.staticTexts[title1].swipeRight()
-		default:
-			app.swipeLeft()
+			app.staticTexts[title1].swipeLeft()
 			
+			cardPandemicRadatInfoScreenTest(title0)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift:258]:
```suggestion
			cardPandemicRadarInfoScreenTest(title0)
```

File: src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift [hunk 5]
```diff
@@ -221,10 +253,14 @@ class ENAUITests_01b_Statistics: CWATestCase {
...
+			app.staticTexts[title2].swipeRight()
+		default:
+			cardPandemicRadatInfoScreenTest(title0)
```
Comment (naveeddotio) [inline src/xcode/ENA/ENAUITests/Home/ENAUITests_01b_Statistics.swift:261]:
```suggestion
			cardPandemicRadarInfoScreenTest(title0)
```

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Small things, rest looks good to me

**Review by flxschmidt — APPROVED** [review]
One suggestion if you want. But already approved :)

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift [hunk 6]
```diff
@@ -27,6 +27,7 @@ class HomeStatisticsCellModel {
 					.compactMap { cardID in
 						statistics.linkCards.first { $0.header.cardID == cardID }
 					}
+				self?.setupMockLinkCards()
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsCellModel.swift:30]: Can you please move this down to line 47 under #if DEBUG? 

if isUITesting {
        self?.setupMockLinkCards()
}

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**